packages/node_modules/@webex/plugin-meetings/src/locus-info/embeddedAppsUtils.js

```diff
@@ -0,0 +1,33 @@
+import {EMBEDDED_APP_TYPES} from '../constants';
+
+const EmbeddedAppsUtils = {};
+
+const SLIDO_REGEX = /.sli.do\//;
+
+/**
+ * Parse the relevant values that we care about
+ * @param {Object} embeddedApp - raw embedded app obejct
+ * @returns {Object} parsedObject - parsed embedded app object
+ */
+EmbeddedAppsUtils.parseApp = (embeddedApp) => {
+  const parsedApp = {...embeddedApp};
+
+  parsedApp.type = EMBEDDED_APP_TYPES.OTHER;
+  const url = parsedApp.instanceInfo?.appInstanceUrl;
+
+  if (url && url.match(SLIDO_REGEX)) {
+    parsedApp.type = EMBEDDED_APP_TYPES.SLIDO;
```

Comment:
I don't see the reason why we need to differentiate slido from other apps - isn't it just another app?

Reply:
even in cantina - i don't think we should give special values to slido - if we trigger sdk for an embedded app, we just show them and hide them irrespective of which app is run

Reply:
This was specifically requested by @arun3528

Reply:
We are going to have to differentiate between app types because integration with each type of app will be different.  Cantina will need to ignore anything other than slido, so Arun wanted the logic to go in the SDK instead of the client app.  More types would be added as we support them.  I'm fine either way, though, so you and Arun can fight it out.  :)

Reply:
okie, I will confirm with Arun on this.

Reply:
Slido alone might have some preference I guess - as we might be showing extra buttons at the bottom